rest-server/src/controllers/job.js

```diff
@@ -56,14 +56,8 @@ const list = (req, res) => {
         error: 'GetJobListError',
         message: 'get job list error'
       });
-    } else if (jobList === undefined || jobList.length === 0) {
-      logger.warn('list jobs error, no job found');
-      return res.status(500).json({
-        error: 'JobListNotFound',
-        message: 'could not find job list'
-      });
     } else {
-      return res.json(jobList);
+      return res.status(200).json(jobList);
```

Comment:
what if error occurs in launcher webservice?

Reply:
I thought all errors have been caught by your above code. Then what status code we return do you suggest?

Reply:
you deleted an `if else` branch so the last `else` branch includes errors

Reply:
Thanks for pointing it out. Must be a mistake.